tests/terratest/functions/set/setAKS.go

```diff
@@ -0,0 +1,105 @@
+package functions
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/hashicorp/hcl/v2/hclsyntax"
+	"github.com/hashicorp/hcl/v2/hclwrite"
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/pkg/config"
+	format "github.com/rancher/rancher/tests/terratest/functions/format"
+	"github.com/rancher/rancher/tests/v2/validation/terratest"
+	"github.com/zclconf/go-cty/cty"
+)
+
+func SetAKS(k8sVersion string, nodePools []terratest.Nodepool, file *os.File) (done bool, err error) {
+	rancherConfig := new(rancher.Config)
+	config.LoadConfig("rancher", rancherConfig)
+
+	terraformConfig := new(terratest.TerraformConfig)
+	config.LoadConfig("terraform", terraformConfig)
+
+	// create new empty hcl file object
```

Comment:
nit; I would say these comments aren't necessary to have.

Reply:
I will add logging forsure, but I don't see the benefit in removing these comments.  Anyone referencing the code will save time being able to scan the comments for the appropriate section, and it doesn't cause clutter IMO.  As we grow in features, we will likely need to add new fields to these blocks for future feature support, I see them being beneficial in the long run.

I don't mind removing them, but I think they are helpful.  

Is the team consensus to remove these comments?

Reply:
Ultimately, I would be fine with the team consensus, but I personally think having both is redundant. @slickwarren @igomez06 what do you think?

Reply:
this comment sounds like a log. So as a maintainer, I should be able to grep the log and understand what the following function is doing. Comments should really be to clarify something non-intuitive in the code that the maintainer wouldn't be able to understand by skimming the following function. If it is obvious what it is, then a comment is redundant. tldr; code clarity -> comment, functional clarity -> log (typically, both aren't used in the same spot)